packages/core/src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -58,7 +58,7 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
                 const payment = this._stripeElements?.getElement(StripeStringConstants.PAYMENT);
                 if (payment) {
                     let error;
-                    await this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(`${gatewayId}?method=${methodId}`)).catch(err => error = err);
+                    await this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(gatewayId, {params: {method: methodId}})).catch(err => error = err);
```

Comment:
@BC-AOrtiz As discussed with Jose, could you review stripe UPE strategy when you have time? Thanks